client/js/lounge.js

```diff
@@ -444,9 +444,16 @@ $(function() {
 		}
 	});
 
-	sidebar.on("click", ".close", function() {
+	function closeChan(itemData) {
```

Comment:
Call this `target`. At this point in the code (i.e. not in generic `contextMenuActions` actions anymore), it's fine to re-specialize and makes the code easier to follow.